static/templates/stream_types.hbs

```diff
@@ -4,12 +4,14 @@
         {{> help_link_widget link="/help/stream-permissions" }}
     </h4>
     {{#each stream_privacy_policy_values}}
-    <li>
-        <label class="radio">
-            <input type="radio" name="privacy" value="{{ this.code }}" {{#if (eq this.code ../stream_privacy_policy) }}checked{{/if}} />
```

Comment:
not sure if the condition inside this statement on `checked` should be removed. 
if it can be removed what purpose did it serve earlier?

Reply:
Hmm, now that I think of it, this check is necessary. I added it back and made requisite changes in js